Security/centurylink-cloud-guide-to-vormetric-dsm.md

```diff
@@ -1,5 +1,5 @@
 {{{
-  "title": "CenturyLink Cloud Guide to Vormetric DSM",
+  "title": "IPS - CenturyLink Cloud Guide to Vormetric DSM",
```

Comment:
I thought it was. Sorry, I figured everything in that folder would be our product.